cset-workflow/meta/rose-meta.conf

```diff
@@ -115,697 +232,1353 @@ sort-key=met1
+        template variables=m10_preprocessing: this >= 10;
+        template variables=m10_preprocessing_recipe: this >= 10;
 
 
 ################################################################################
-# Data and Cycling
+# Model definitions. Repeat section for each desired model.
 ################################################################################
 
-[template variables=CSET_INITIAL_CYCLE_POINT]
-ns=Data and Cycling
-description=Datetime of initial cycle point.
-help=This should be an ISO 8601 datetime string indicating when you want the
-    cycling component of CSET to begin. E.g. 2000-01-01T00:00Z. Ensure that it
-    is consistent with your data's starting validity time. If not then a warning
-    that cubes can not be loaded is raised.
+# [template variables=m??_name]
+# ns=Models and Cases/Model ??
+# title=Model name
+# description=A friendly name for the model.
+# help=A recognisable name for this particular model. This is carried through to
+#      the output webpage, and identifies the model. As it is only used for
+#      display purposes it can be any string.
+# type=quoted
+# compulsory=true
+# sort-key=a0
+
+# # Model ??
+# [template variables=m??_data_source]
+# ns=Models and Cases/Model ??
+# title=Data source
+# description=From whence to retrieve the forecast.
+# help=System from which to retrieve the forecast data. Filesystem is generic,
+#     while the others are site-specific.
+# values="filesystem", "mass"
+# value-titles=Filesystem, MASS
+# compulsory=true
+# sort-key=a1
+
+# [template variables=m??_data_path]
+# ns=Models and Cases/Model ??
+# title=Data path
+# description=The path to the forecast.
+# help=Full path (including file name) to the forecast data on your chosen storage
+#     system. Can contain wildcards. No quotation marks required in rose edit, as
+#     it is already quoted there.
+#
+#     strftime format strings are supported, and will be replaced with the
+#     desired case study date or trial time. E.g: `/data/%Y%m%d/model1/*.nc`
+#
+#     For forecast lead time, %N can be used which is replaced by the 3 digit
+#     lead time in hours. The regular placeholders are still usable, and refer
+#     to the initiation time E.g: `/data/case_%Y%m%d/model1_%N.nc`
+#
+#     Commonly useful placeholders:
+#     %Y: Year, e.g: 2024
+#     %m: Month, e.g: 12
+#     %d: Day, e.g: 31
+#     %H: Hour, e.g: 23
+#     %M: Minute, e.g: 59
+#     %N: Forecast lead time in hours, e.g: 012
+#
+#     For more on strftime format strings, see: https://strftime.org
+# type=quoted
+# compulsory=true
+# sort-key=a2
+
+# [template variables=m??_date_type]
+# ns=Models and Cases/Model ??
+# title=Date placeholder type
+# description=Type of date templated into the data path.
+# help=The type of date templated into the data path. Affects interpretation of
+#     strftime placeholders in the data path.
+
+#     Validity time is when the data is predicting for.
```

Comment:
Validity time is the date and time the simulated or predicted data are valid for. This time is usually used to compare against observations.

Reply:
Fixed in 1189137ae5d9dda064322318d4f4fc125f0d41b4